osu.Game/Overlays/Settings/SettingsFooter.cs

```diff
@@ -108,6 +127,19 @@ private void load(ChangelogOverlay? changelog)
                     Colour = DebugUtils.IsDebugBuild ? colours.Red : Color4.White,
                 });
             }
+
+            public MenuItem[] ContextMenuItems
+            {
+                get
+                {
+                    List<MenuItem> menuItems = new List<MenuItem>()
+                    {
+                        new OsuMenuItem("Copy Version", MenuItemType.Standard, () => clipboard.SetText(version))
```

Comment:
Should probably show an OSD like this:
https://github.com/ppy/osu/blob/01828931a211fd4de31cacedbd73a247c1e22bd2/osu.Game/OsuGame.cs#L522-L526

Either make another method in `OsuGame` for non-links, or remove the url/link wording and make it broad.